src/components/functions/showPlayer.tsx

```diff
@@ -0,0 +1,4 @@
+export default function showPlayer(player:HeroInfo) {
```

Comment:
not a good name

Reply:
Maybe have 2 separate functions one for setCurrentPlayer and one for Navigate to player screen.
Also I think it is better to use ReactContext for current player you don't need it in the localStorage.